corehq/apps/app_manager/static/app_manager/js/releases.js

```diff
@@ -161,6 +161,7 @@ hqDefine('app_manager/js/releases.js', function () {
             ga_track_event('App Manager', 'Deploy Button', self.id());
             analytics.workflow('Clicked Deploy');
             $.post(releasesMain.options.urls.hubspot_click_deploy);
+            self.get_short_odk_url();
```

Comment:
I'm not sure I understand why this is necessary.

Also, and maybe this is related, I noticed `get_short_odk_url` is just a function and not a computed observable, so it won't get re-run when, say, `self.include_media()`, which it depends on, changes. You can change that by using

```javascript
self.get_short_odk_url = ko.computed(function() {
};)
```
instead.